package.json

```diff
@@ -25,6 +25,7 @@
     "@atlaskit/icon": "10.0.0",
     "@atlaskit/inline-dialog": "5.0.2",
     "@atlaskit/inline-message": "3.0.1",
+    "@atlaskit/lozenge": "3.4.2",
```

Comment:
If I'm not mistaken, whenever we change package.json we should also commit the package-lock changes now.

Reply:
@hristoterezov can you confirm?

Reply:
I'll update it after another update to it for fixing IE11 and adding safari+adapter gets merged in. I'll rebase, npm install, and push the package-lock change then.

Reply:
package-lock has been updated now.